optimize/self-managed/optimize-deployment/migration-update/translation-diffs/differences_localization_311_312.diff

```diff
@@ -0,0 +1,244 @@
+       "runningProcesses": "Which processes are running?",
+@@ -978,14 +953,15 @@
+     "contactManager": "Please contact the collection manager to create new Alerts",
+     "inactiveStatus": "Alert inactive",
+     "activateInfo": "To active add email or target system",
+-    "existingAlerts": "Existing alerts",
+     "updated": "\"{name}\" updated.",
+     "removed": "\"{name}\" removed."
+   },
+   "analysis": {
+     "branchAnalysis": "Branch Analysis",
+-    "outlier": {
+-      "label": "Outlier Analysis",
++    "task": {
++      "label": "Task Analysis",
++      "totalTaskInstances": "Total task instances: {count}",
++      "result": "Result: {count} Process instances do match these criteria",
+       "totalInstances": "Total Instances",
+       "tooltip": {
+         "outlier": {
+@@ -995,17 +971,23 @@
+         "tookDuration": {
+           "singular": "took a duration of",
+           "plural": "took a duration of"
+-        }
++        },
++        "zScore": "Outliers are calculated based on Z-score. Z-score is a statistical measurement that describes a value's relationship to the mean of a group of values. Z-score is measured in terms of standard deviations from the mean."
+       },
+       "notFound": "No outliers found for the selected process",
+-      "onlyCompletedHint": "Note: The outlier analysis is performed on completed flow nodes only",
++      "onlyCompletedHint": "Note: The task analysis is performed on completed flow nodes only",
+       "info": "Heatmap displays the incidence of higher outliers based on duration. Calculated by z-score.",
+       "tooltipText": {
+         "singular": "{count} instance took {percentage}% longer than average.",
+         "plural": "{count} instances took {percentage}% longer than average."
+       },
++      "table": {
++        "heading": "Outlying tasks",
++        "taskName": "Task Name",
++        "outliers": "Outliers"
++      },
+       "detailsModal": {
+-        "title": "Outliers of {name} Flow Node",
++        "title": "Results for task \"{name}\"",
+         "durationChart": "Duration Distribution chart",
+         "variablesTable": "Common Significant Variables Table",
+         "axisLabels": {
+@@ -1077,6 +1059,7 @@
+     "disabled": "disabled",
+     "and": "and",
+     "or": "or",
++    "on": "On",
+     "off": "Off",
+     "none": "None",
+     "all": "All",
+@@ -1110,6 +1093,11 @@
+     "less": "Less",
+     "configure": "Configure",
+     "new": "New",
++    "error": "Error",
++    "closeError": "Close error notification",
++    "units": "Units",
++    "link": "Link",
++    "details": "Details",
+     "process": {
+       "label": "Process",
+       "label-plural": "Processes"
+@@ -1285,6 +1273,7 @@
+         "duration": "Duration",
+         "instanceDuration": "Instance Duration",
+         "processInstanceDuration": "Process Instance Duration",
++        "dur": "Flow Node Duration",
+         "flowNodeDuration": "Flow Node Duration",
```

Comment:
is this right? Having two fields here with the same value feels wrong. Are they being used in FE?

Reply:
I believe this is used by the filter type